api/src/main/java/io/opentelemetry/trace/SpanId.java

```diff
@@ -119,23 +93,8 @@ public void copyBytesTo(byte[] dest, int destOffset) {
    *     srcOffset}.
    * @since 0.1.0
    */
-  public static SpanId fromLowerBase16(CharSequence src, int srcOffset) {
-    Utils.checkNotNull(src, "src");
-    return new SpanId(BigendianEncoding.longFromBase16String(src, srcOffset));
-  }
-
-  /**
-   * Copies the lowercase base16 representations of the {@code SpanId} into the {@code dest}
-   * beginning at the {@code destOffset} offset.
-   *
-   * @param dest the destination buffer.
-   * @param destOffset the starting offset in the destination buffer.
-   * @throws IndexOutOfBoundsException if {@code destOffset + 2 * SpanId.getSize()} is greater than
-   *     {@code dest.length}.
-   * @since 0.1.0
-   */
-  public void copyLowerBase16To(char[] dest, int destOffset) {
-    BigendianEncoding.longToBase16String(id, dest, destOffset);
+  public static byte[] bytesFromLowerBase16(String src, int srcOffset) {
```

Comment:
Same, use Hex if you decided to use that in the SC

Reply:
agreed. 👍

Reply:
done